### PR TITLE
Style: fix ~1/3 of the PEP8 warnings in tests

### PR DESCRIPTION
**Pull request by thsutton**
I got through this much before I saw your comment about leaving the test suite.

### CODE REVIEW

File: aristotle_mdr/tests/apps/extension_test/models.py [hunk 1]
```diff
@@ -1,29 +1,33 @@
 from __future__ import unicode_literals
 
-# Start of the question model
```
Comment (LegoStormtroopr) [inline aristotle_mdr/tests/apps/extension_test/models.py]: The comments in this file are use for sphinx documentation, so they need to be there. Can you add them back in?

Comment (thsutton) [inline aristotle_mdr/tests/apps/extension_test/models.py]: Done

**Comment by LegoStormtroopr** [comment]
No stress, this is all looking good. I had a quick look last night and it seems ok.
Keep going with this if you want, I'll just go through this a little slower and more careful just because of their nature.

File: aristotle_mdr/tests/apps/text_download_test/models.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Required for Django 1.6.x
```
Comment (LegoStormtroopr) [inline aristotle_mdr/tests/apps/text_download_test/models.py]: this file was only needed for django<1.7. Can you remove this on the next commit if the tests still pass, so we can just drop it as we only support 1.8+

**Comment by LegoStormtroopr** [comment]
Looks great!

**Pull request merged**